README.md

```diff
@@ -0,0 +1,25 @@
+# Geophoto
+
+## What does this software do:
+1) Takes folder which contains huge amount of jpeg images - ```<source_folder_path>```
+2) Takes every image in order which contains geodata, images that do not contain geodata will be skipped
+3) Writes geodata to 'dataset.geojson' file, it will be automatically generated in ```<output_folder_path>``` directory
+4) Resizes chosen images, depending on the value you gave it in ```<size>``` parameter and saves them to ```<output_folder_path>```, this is made for comfortable preview purposes
+5) As a result you will have: ```<output_folder_path>```, which contains all required photos and .geojson file with metadata
+
+### Install:
+pip package manager is not available for this project yet, so you need to clone this repository:
+```
+$ git clone git@github.com:designunit/geophoto.git
```

Comment:
Replace with *https* url.

If I remember correctly, cloning with ssh url will require public key associated with this repository. Most likely, end user does not have one.

Reply:
I will test it and make changes depending on the result.